tealium-xcframeworks/TealiumFirebase.xcframework/ios-arm64_x86_64-simulator/TealiumFirebase.framework/Headers/FirebaseConstants.swift

```diff
@@ -0,0 +1,51 @@
+//
```

Comment:
Are these files always tracked? Should they be?

Reply:
They were tracked by mistake